### PR TITLE
[FIX] website_sale: add message to translation

### PR DESCRIPTION
**Pull request by AH-Yussef**
### Steps to reproduce:
- install the e-commerce module
- in General settings, add and switch French -or any- language
- Go to website
- Under `e-commerce > products` in the menu bar
- click on any product
- click on go to website smart button
- click on edit button to edit the website page
- choose any block to insert, You can notice the message **DROP BUILDING BLOCKS HERE TO MAKE THEM AVAILABLE ACROSS ALL PRODUCTS** stays in English

### Investigation
- The message is not add to `website_sale.pot` file, as the message string passed to an attribute not an actual string inside a tag.

opw-3573881

### CODE REVIEW

**Review by nle-odoo — APPROVED** [review]
looks good to me

File: addons/website_sale/views/templates.xml [hunk 1]
```diff
@@ -567,8 +567,9 @@
 
         <t t-call="website.layout">
             <t t-set="additional_title" t-value="product.name" />
+            <t t-set='editor_msg'>DROP BUILDING BLOCKS HERE TO MAKE THEM AVAILABLE ACROSS ALL PRODUCTS</t>
```
Comment (Feyensv) [inline addons/website_sale/views/templates.xml:570]:
```suggestion
            <t t-set="editor_msg">DROP BUILDING BLOCKS HERE TO MAKE THEM AVAILABLE ACROSS ALL PRODUCTS</t>
```
Please prefer the use of double quoted strings in xml files ;)

**Review by Feyensv — APPROVED** [review]
@robodoo delegate+
LGTM, but this should probably be handled directly in the translation engine
cc: @odoo/i18n

**Comment by mart-e** [comment]
>  LGTM, but this should probably be handled directly in the translation engine

FYI this is defined here

https://github.com/odoo-dev/odoo/blob/f52b652149a539e50999fe7a58614254ae200ab1/odoo/tools/translate.py#L157-L161

The alternative is to use one of these already defined element like `help` or `data-tooltip` instead of using yet another custom attribute

**Comment by Feyensv** [comment]
Might be worth fixing all occurrences of `data-editor-message` at once by adding the attribute to the list no?
This logic doesn't seem recent so it's a bit unexpected that it was never considered for translations :thinking:.

**Pull request closed without merging**